core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -163,12 +168,12 @@ public void close() {
    */
   @Nullable
   private static PageId getPageIdFromKey(byte[] key) {
-    if (key.length != KEY_LEN) {
+    if (key.length < Long.BYTES) {
       return null;
     }
     ByteBuffer buf = ByteBuffer.wrap(key);
-    long fileId = buf.getLong();
     long pageIndex = buf.getLong();
+    String fileId = Charset.defaultCharset().decode(buf).toString();
```

Comment:
Do you know how much more expensive this is compared to `getLong`?

Reply:
Probably a lot more expensive. Not sure how much it impact the overall performance though.

Reply:
yeah something we can keep in mind if performance is an issue